smart-contracts/contracts/AgroXchangePriceFeed.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {ConfirmedOwner} from "@chainlink/contracts/src/v0.8/shared/access/ConfirmedOwner.sol";
+import {ChainlinkClient} from "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+import {Chainlink} from "@chainlink/contracts/src/v0.8/Chainlink.sol";
+import {LinkTokenInterface} from "@chainlink/contracts/src/v0.8/shared/interfaces/LinkTokenInterface.sol";
+
+contract AgroXchangePriceFeed is ChainlinkClient, ConfirmedOwner {
+    using Chainlink for Chainlink.Request;
+
+    address private oracleAddress;
+    bytes32 private jobId;
+    uint256 private fee;
+    uint256 public response;
+
+    constructor(
+        address _owner,
```

Comment:
why not use msg.sender here instead of requesting for owner separately

Reply:
Being more explicit.